app/connectors/OutgoingRequestFilter.scala

```diff
@@ -36,9 +35,9 @@ object OutgoingRequestFilter {
     requestHeader.headers.headers.filter(x => CustomUpstreamHeaders.contains(x._1))
   }
 
-  def enforceAuthHeaderCarrier()(implicit requestHeader: RequestHeader, headerCarrier: HeaderCarrier): HeaderCarrier = {
+  def enforceAuthHeaderCarrier(eisBearerToken: String)(implicit requestHeader: RequestHeader, headerCarrier: HeaderCarrier): HeaderCarrier = {
```

Comment:
Can the bearer not just be added on in the connector rather then passing it through here?